OpenTrons/opentrons-api
# feat(protocol-designer): wireup wasteChute commands

closes [RAUT-767](https://opentrons.atlassian.net/browse/RAUT-767)

# Overview

This PR updates the pipetting compound commands to check the dropTipLocation. If the location is included in additionalEquipment, wasteChuteCommandsUtil is called with the correct arguments needed. This is plugged into returnTip and generateRobotStateTimeline.

We also update the robot tip state when the commands get emitted by updating forDropTipLocation.

# Test Plan

- write test to verify that returned commands from replaceTip called at wasteChute return a moveToAddressableArea, dropTipInPlace, and pickUpTip
- check tip state to ensure replaceTip at waste chute updates used tips
- verify that we no-op moveToAddressableArea and dropTipInPlace if no tip attached

# Changelog

- 

# Review requests

@jerader per collaboration

# Risk assessment



[RAUT-767]: https://opentrons.atlassian.net/browse/RAUT-767?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
